Firebase/Core/FIRLogger.m

```diff
@@ -14,16 +14,10 @@
 
 #import "Private/FIRLogger.h"
 
-#import "FIRLoggerLevel.h"
+#import <FirebaseCore/FIRLoggerLevel.h>
+#import <GoogleUtilities/GULAppEnvironmentUtil.h>
+#import <GoogleUtilities/GULLogger.h>
```

Comment:
Nit: Should have vertical whitespace between the framework imports and file import.